configurations/es5.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    extends: [
+        '../rules/eslint/best-practices/on.js',
+        '../rules/eslint/possible-errors/on.js',
+        '../rules/eslint/ecma-script-6/off.js',
+        '../rules/eslint/node-js-and-common-js/off.js',
+        '../rules/eslint/strict-mode/on.js',
+        '../rules/eslint/stylistic-issues/on.js',
+        '../rules/eslint/variables/on.js',
+        '../rules/eslint/deprecated/off.js'
```

Comment:
Why do we require in turned off rules here?
`'../rules/eslint/deprecated/off.js'`